src/app-host/app-host.js

```diff
@@ -57,38 +100,6 @@ function getCordova() {
     return cordova;
 }
 
```

Comment:
Thanks for catching this! It was behaving weirdly on me, but I didn't get to actually root-cause it.